01week/javascripting/introduction.js

```diff
@@ -1 +1,78 @@
-console.log('hello');
+'use strict';
+
+// Write a JavaScript program to display the current day and time.
+
+function currentDateTime(){
+  console.log(new Date());
+}
+currentDateTime();
+
+
+// Write a JavaScript program to convert a number to a string.
+
+function numberToString(num){
+  console.log(num.toString());
```

Comment:
The logic is right, however the function call is a little off. Your function should return the new date, and you should call the function in a console.log to display.